### PR TITLE
VZE: Map drags when dragging cursor in the CrashQAMap geocoder address bar

### LINKED ISSUE #168: VZE: Map drags when dragging cursor in the CrashQAMap geocoder address bar
When hovering over the address bar of the geocoder in the CrashQAMap component, clicking and dragging with the cursor moves the map below. The issue is possibly related to the z-index of the geocoder input or event bubbling. This bug is also present in the example from the creator of the [library](https://github.com/SamSamskies/react-map-gl-geocoder) (https://codesandbox.io/s/l7p179qr6m).

64653067-0a97b300-d3eb-11e9-9da0-12a680029318

### PR DESCRIPTION
**Pull request by mddilley**
Closes #168

### CODE REVIEW

**Comment by mddilley** [comment]
This PR fixes an issue where double-clicking or dragging over the geocoder or map style controls propagated events to the `<CrashQAMap />` below. Invaluable advice from @SamSamskies (https://github.com/SamSamskies/react-map-gl-geocoder/issues/53) led to a quick fix of adding a custom `MapController` that ignores these events for these specific control elements (https://github.com/uber/react-map-gl/blob/master/docs/advanced/custom-map-controller.md). 

The string in the geocoder control can now be selected by clicking and dragging without causing changes in the map below.

File: atd-vze/src/views/Crashes/Maps/customGeocoderMapController.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { MapController } from "react-map-gl";
+
+export class CustomGeocoderMapController extends MapController {
+  _onPan(event) {
```
Comment (mateoclarke) [inline atd-vze/src/views/Crashes/Maps/customGeocoderMapController.js]: so these methods overwrite the same `MapController` methods, I'm assuming?

Comment (mateoclarke) [reply atd-vze/src/views/Crashes/Maps/customGeocoderMapController.js]: now digging into your links, I'm seeing that's exactly what it does. Can you add a comment to this file that gives a brief explanation and links to [the examples](https://github.com/uber/react-map-gl/blob/master/docs/advanced/custom-map-controller.md)?

**Comment by mateoclarke** [comment]
It's super cool that you were able to reach out to the library maintainer and get a helpful pointer. We should send them some Tiff's Treats 🍪

File: atd-vze/src/views/Crashes/Maps/customGeocoderMapController.js (hunk 1 above)
Comment (mddilley) [reply atd-vze/src/views/Crashes/Maps/customGeocoderMapController.js]: Great idea - I added in some documentation so it is easier to see what is going on. I'm sure that we will see this come up again with this library too.

**Pull request merged**